applications/StructuralMechanicsApplication/custom_conditions/small_displacement_line_load_condition.cpp

```diff
@@ -143,8 +143,8 @@ void SmallDisplacementLineLoadCondition<TDim>::CalculateAll(
     Vector pressure_on_nodes = ZeroVector( number_of_nodes );
 
     // Pressure applied to the element itself
-    double pressure_on_condition = 0.0;
-    if (TDim == 2) {
+    if constexpr (TDim == 2) {
+        double pressure_on_condition = 0.0;
```

Comment:
Why is this inside the if now? (not familiar with the code, but in some other similar codes you left it ouside)

Reply:
Because otherwise clang complains (because unused variable if TDim is different from 2). The variable is only used in the if anyway